### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by kgaugelomokgalaka**
added a chat bot 
changed the styling on the education page

### CODE REVIEW

**Comment by neoThobela** [comment]
the code is easy to read and understand

**Comment by NthabisengLekolwana** [comment]
I think your chatbot 's look goes with your website and is efficient.

**Comment by XolaniMba** [comment]
i like the use of color and the predefined responses

File: EDUCATION.HTML [hunk 1]
```diff
@@ -37,7 +37,7 @@
 <br>
 <br>
 <br>
```
Comment (XolaniMba) [inline EDUCATION.HTML:39]: Hi i see to many break (br) lines Try adding padding

File: script.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+function getBotResponse(input) {
+    const responses = {
+        "hi": "Hello! How can I assist you today?",
+        "hello": "Hi there!",
+        "how are you": "I'm here to help!",
+        "name": "kgaugelo Helen Mokgalaka",
+         "education": "Admission to bachelors degree from All saints and bachelors degree in information technology in business systems from Rosebank college!",
+          "skills": "javaScript, HTML, CSS, SQL, communication skills , team work skills creativity skils!",
+          "view cv": "kgaugelo\MOKGALAKA KGAUGELO RESUME .pdf",
+    };
```
Comment (XolaniMba) [inline script.js]: Hi, try adding some more responses

Comment (kgaugelomokgalaka) [reply script.js]: alright i have removed the last responce its the one that has the least responds

File: EDUCATION.HTML (hunk 1 above)
Comment (XolaniMba) [reply EDUCATION.HTML:39]: Okay i see the above changes have been made. Thanks

**Review by XolaniMba — APPROVED** [review]

**Review by XolaniMba — COMMENTED** [review]
please change your styling for the box fix the margin to be specific

**Comment by kgaugelomokgalaka** [comment]
> please change your styling for the box fix the margin to be specific

@XolaniMba i have changed the styles please check if it looks good now

**Comment by XolaniMba** [comment]
> > please change your styling for the box fix the margin to be specific
> 
> @XolaniMba i have changed the styles please check if it looks good now

okay I will check it

**Review by XolaniMba — COMMENTED** [review]
your code looks all good now 
i will approve it

**Pull request merged**